cwms_radar_api/src/main/java/cwms/radar/data/dto/rating/AbstractRatingMetadata.java

```diff
@@ -15,23 +19,31 @@
                 ExpressionRating.class,
                 UsgsStreamRating.class
         },
-        discriminatorProperty = "ratingType",
+        discriminatorProperty = "rating-type",
         discriminatorMapping = {
                 @DiscriminatorMapping(value = TableRating.RATING_TYPE, schema = TableRating.class),
```

Comment:
I wonder if this will work well for the text timeseries. I suspect so and am glad you found it.